api/src/libs/migrateBusinessesFromUsers.ts

```diff
@@ -0,0 +1,53 @@
+import { BusinessesDataClient, UserDataClient } from "@domain/types";
+import { LogWriterType } from "@libs/logWriter";
+import { UserData } from "@shared/userData";
+
+export const migrateBusinessesFromUsers = async (
+  userDataClient: UserDataClient,
+  businessesDataClient: BusinessesDataClient,
+  logger: LogWriterType
+): Promise<void> => {
+  const processBatch = async (users: UserData[]): Promise<void> => {
+    for (const userData of users) {
+      const businessesData = userData.businesses;
+
+      if (businessesData && Object.keys(businessesData).length > 0) {
+        for (const businessId in businessesData) {
+          const businessData = businessesData[businessId];
+          const existingBusiness = await businessesDataClient.get(businessId);
+          const currentVersion = existingBusiness ? existingBusiness.version + 1 : 1;
+
+          await businessesDataClient.put({
```

Comment:
Within `businessesDataClient`, we are no longer running the migration fn, but we are in `userDataClient`. Doesn't this mean the data is not in sync?

Reply:
We might store V99 in the business table and v147 (current) in the users table?

Reply:
This isn't an issue if every user has migrated to the current version (I forgot if we've previously done that)

Reply:
Yh. This would no longer exist. Since we have the lambda function that migrates all the users to the latest version.